CMakeLists.txt

```diff
@@ -83,6 +83,8 @@ if(MUSL_OPTIMIZED_BUILD)
 	set(SYSDIG_MUSL_FLAGS "-static -Os")
 endif()
 
+set(WITH_CHISEL ON CACHE BOOL "")
```

Comment:
Let's get rid of this one, chisels are a core feature of sysdig and I see no reason right now to have them conditional